web-ui/src/main/resources/catalog/templates/top-toolbar.html

```diff
@@ -186,7 +186,7 @@
           </li>
           <li class="dropdown-header hidden-xs" role="menuitem" translate>profile</li>
           <li class="hidden-xs" role="menuitem">
-            <a data-gn-active-tb-item="{{gnCfg.mods.admin.appUrl}}#/organization/users?userOrGroup={{user.username}}">{{user.profile | lowercase | translate}}</a>
+            <a style="text-transform: lowercase" data-gn-active-tb-item="{{gnCfg.mods.admin.appUrl}}#/organization/users?userOrGroup={{user.username}}">{{user.profile | translate}}</a>
```

Comment:
This is creating the following link, the `href` element doesn't have the `username`, probably was like that previously also, so the users page is displayed, but not with the current user information:

```
<a data-gn-active-tb-item="../../{{node}}/{{lang}}/admin.console#/organization/users?userOrGroup=reviewer" title="User details" class="dropdown-toggle gn-menuitem-xs" role="button" aria-expanded="false" href="../../srv/eng/admin.console#/organization/users?userOrGroup="> 
```